### DISCUSSION TITLE
Generate swagger only for Upstream URL

### DISCUSSION THREAD

**Discussion by ghost** [Burgyn/MMLib.SwaggerForOcelot#278] [Q&A]
Hi,

I have a requirement where I want to show only the Swagger docs for my Upstream URL

Sample Ocelot code

```
 {
  "DownstreamPathTemplate": "/project/api/{everything}",
  "DownstreamScheme": "http",
  "UpstreamPathTemplate": "/api/project/{everything}",
}
```
Expected swagger endpoints:

`/api/project/{everything}`

Is any solution available for this scenario?

Any help would be highly appreciated!

Thanks

**Comment by Burgyn** [comment]
Hi @ali-niyas,

I apologize for the confusion. Could you please clarify the current behavior you're experiencing?
